Finjector.Web/ClientApp/src/components/Folders/DeleteFolder.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import { Modal, ModalHeader, ModalBody, ModalFooter } from "reactstrap";
+import { useDeleteFolderMutation } from "../../queries/folderQueries";
+import { useNavigate } from "react-router-dom";
+import FinjectorButton from "../Shared/FinjectorButton";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+
+interface Props {
+  folderId: string;
+}
+
+const DeleteFolder = (props: Props) => {
+  const navigate = useNavigate();
+
+  const [modalOpen, setModalOpen] = useState(false);
+
+  const deleteMutation = useDeleteFolderMutation();
+
+  const toggleModal = () => {
+    setModalOpen(!modalOpen);
+  };
+
+  const handleDelete = () => {
+    deleteMutation.mutate(props.folderId, {
+      onSuccess: () => {
+        navigate("/teams");
+        toggleModal();
```

Comment:
probably better to just `setModalOpen(false)` instead of toggle.  i know it should be open but you want to make sure it is